app/javascript/components/layout/CustomTable.js

```diff
@@ -159,6 +160,7 @@ class CustomTable extends React.Component {
     return (
       <th
         key={field}
+        id={field}
```

Comment:
for fields like `id`, `updated_at`, and `created_at`, won't there be redundant `<th>` elements on the same page because assessments, vaccinations, lab results, etc will all render these?

Reply:
good catch, not sure it's an issue right now because wave didn't flag anything but it definitely will be when the labs table is added: https://github.com/SaraAlert/SaraAlert/pull/1176/commits/712bfcab374e5bb502a8432b8e4be723ea4cbd78

Reply:
For documentation's sake, I tried to change the tests to find by the key which would remove the need for the id.  I was unable to simulate clicking the headers with that route, so this is the way to go